lib/sandbox/sandbox.go

```diff
@@ -99,6 +100,7 @@ type Sandbox struct {
 	portMappings   []*hostport.PortMapping
 	created        bool
 	stopped        bool
+	stopMutex      sync.RWMutex
```

Comment:
Can this be just a Mutex interface? To avoid pointers and references around

Reply:
I'll need to use both the read and write locks.  I preferred to add one function instead of adding four, but I agree it is more risk prone, so I can change it if you prefer